style.css

```diff
@@ -321,6 +344,133 @@ textarea {
   transform: translateX(5px);
 }
 
+/* popup window section */
+
+.container {
+  padding: 16px;
+  display: flex;
+  gap: 12px;
+  flex-direction: column;
+  justify-content: flex-start;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transition: 300ms ease-in-out;
+  transform: translate(-50%, -50%) scale(0);
+  background-color: white;
+  border: 1px solid #efe1e6;
+  border-radius: 16px;
+  z-index: 10;
+  max-width: 343px;
+  align-items: flex-start;
+}
```

Comment:
Kindly consider adding size limitations to your popup windows and giving them [`overflow` properties](https://www.w3schools.com/css/css_overflow.asp) instead :wink:, they seem to be going off the screen in my PC :mag: 
![image](https://user-images.githubusercontent.com/78335447/175752249-a7f1a4d1-8c17-435a-bd0f-0e46f6796526.png)

![image](https://user-images.githubusercontent.com/78335447/175752265-a2ffde95-a8b7-4bb5-87c1-91f82f318264.png)